CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.7.5
+
+* Update SSM agent to 3.2.1705.0 ([#51])
+
+[#50]: https://github.com/bottlerocket-os/bottlerocket-control-container/pull/51
```

Comment:
There's a typo here, with the label being `#50` but the reference above and actual URL being for PR 51.

Reply:
Nice catch!